packages/@aws-cdk/core/lib/cfn-fn.ts

```diff
@@ -172,6 +172,18 @@ export class Fn {
     return new FnFindInMap(mapName, topLevelKey, secondLevelKey).toString();
   }
 
+  /**
+   * The intrinsic function Fn::Transform specifies a macro to perform custom processing on part of a stack template.
+   * Macros enable you to perform custom processing on templates, from simple actions like find-and-replace operations
+   * to extensive transformations of entire templates.
+   * @param macroName The name of the macro to perform the processing
+   * @param parameters The parameters to be passed to the macro
+   * @returns a token represented as a string
+   */
+  public static transform(macroName: string, parameters: '{[name: string]: any}'): string {
```

Comment:
1. Same comment here:

> The type declaration should be just `{ [name: string]: any }` (spaces), no `'`.

2. Since transforms can [return arbitrary JSON](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-macros.html#template-macros-lambda-interface), this should return an `IResolvable`, not a `string`.